src/Akka.Logger.Serilog/SerilogLoggingAdapter.cs

```diff
@@ -7,8 +7,12 @@
 
 namespace Akka.Logger.Serilog
 {
-    public class SerilogLoggingAdapter : ILoggingAdapter
+    public class SerilogLoggingAdapter : BusLogging
```

Comment:
Instead of wrapping an external `ILoggingAdapter` we're just going to be a first-party `BugLogging` adapter, which gives us the ability to choose how we format all of our events right out of the gate. If we wrapped an external `ILoggingAdapter` then we're totally dependent on _them_ using the `SerilogLogMessageFormatter` in order for us to do our job without causing #43 when using Serilog-style semantic logs.